### DISCUSSION TITLE
package default index file not found

### DISCUSSION THREAD

**Discussion by njngwn** [naverz/zepeto-studio-kor#952] [5. Q&A]
안녕하세요. 얼마 전까지는 빌드가 잘 되었었는데 오늘 계속 안되어서 여쭤봅니다.
Play on Zepeto 버튼을 눌렀을 때, System.Exception: package default index file not found. 에러가 뜨는데 혹시 해결 방법 아시는 분 계신가요??

![image](https://user-images.githubusercontent.com/48943487/177087371-967d3a10-f9bf-413e-8d57-1c853f0088bd.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 월드 패키지 버전 및 개발 환경이 어떻게 되시나요?

이후에는 위와 같은 간헐적 오류 발생시 진행 상황 및 로드맵 관리를 위하여 이슈 게시판을 통해 등록 부탁 드립니다.
https://github.com/naverz/zepeto-studio-kor/issues

동일한 오류를 겪는 분들께 많은 도움이 될 것입니다. 
감사합니다.

**Reply by njngwn** [reply]
개발 환경은 현재 윈도우에서 작업 중이며
유니티 버전은 2020.3.9f1 이고, 
월드 패키지 버전은 1.4.1 입니다.

**Reply by Zepeto-TechEvangelist** [reply]
넵, 확인해 보겠습니다.

**Reply by Zepeto-TechEvangelist** [reply]
저희 쪽에서는 재현이 되지 않고 있어서 [dl_zepeto_evangelist@naverz-corp.com](mailto:dl_zepeto_evangelist@naverz-corp.com) 메일로
프로젝트 파일을 전달해 주실 수 있을까요?
